.vscode-example/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "prettier.enable": false,
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+      "source.fixAll.eslint": true
+    }
+}
```

Comment:
@futantan if your vscode don't format on save, you can copy vscode-example/settings.json to try, it's work for me

Reply:
https://user-images.githubusercontent.com/6268441/224469317-851a6736-f94c-4063-958f-048d2830b408.mp4

any other steps needed?

Reply:
i try as your code, the format is working, do you install the eslint vscode extension or you can paste your vscode settings.json here

Reply:
or try restart the vscode

Reply:
dont forget run pnpm i

Reply:
all tried, not working...

Uploading CleanShot 2023-03-11 at 14.59.37.mp4…

Reply:
ok i got it, i will  add prettier back

Reply:
thank you! and I will revert the PR first  so that won't block my work and merge another one